packages/form-js-viewer/src/render/hooks/useRestrictedMultiSelectValues.js

```diff
@@ -0,0 +1,32 @@
+import { useEffect } from 'preact/hooks';
```

Comment:
I see no test added for this new behavior. Did I miss something?

Reply:
There be tests now